restapi/src/main/java/io/stargate/web/docsapi/service/JsonConverterService.java

```diff
@@ -0,0 +1,290 @@
+package io.stargate.web.docsapi.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.*;
+import io.stargate.db.datastore.Row;
+import io.stargate.db.schema.Column;
+import io.stargate.web.docsapi.dao.DocumentDB;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
+public class JsonConverterService {
+  private static final ObjectMapper mapper = new ObjectMapper();
+
+  public ObjectMapper getMapper() {
+    return mapper;
+  }
+
+  // TODO find a place to refactor this to, along with DocumentService#getBooleanFromRow
+  private static Boolean getBooleanFromRow(Row row, String colName, boolean numericBooleans) {
```

Comment:
Would it be cleaner to make `numericBooleans` a instance-level field as opposed to a method parameter?

I guess this is value is fixed in each Stargate process (i.e. it does not change from call to call). WDYT?

Reply:
Yea that would be clearer for sure